Barebox support environment in efi variable. This should be disabled to prevent
overriding our included environment
diff --git a/common/efi/efi.c b/common/efi/efi.c
index 4b42f5d..b1f2a4c 100644
--- a/common/efi/efi.c
+++ b/common/efi/efi.c
@@ -295,6 +295,7 @@ extern char image_base[];
 extern initcall_t __barebox_initcalls_start[], __barebox_early_initcalls_end[],
 		  __barebox_initcalls_end[];
 
+#if 0
 static int efi_init(void)
 {
 	char *env;
@@ -307,6 +308,7 @@ static int efi_init(void)
 	return 0;
 }
 device_initcall(efi_init);
+#endif
 
 /**
  * efi-main - Entry point for EFI images
